.github/workflows/unit-test.yml

```
@@ -2,34 +1,35 @@
-#         path: coverage/coverage-summary.json
-#         title: Unit Test Coverage
-#     - name: Upload coverage to Codecov
-#       uses: codecov/codecov-action@v3
+jobs:
+  unit-test:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup GitHub Action
+      uses: actions/setup-node@v1
```

actionlint

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)